frontend/README.md

```diff
@@ -1,34 +1,20 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+## Setting up dev env
 
-## Getting Started
+1.  Create docker-compose.override.yml and fill in osuApi env vars
 
-First, run the development server:
+1.  Get env vars by creating https://osu.ppy.sh/home/account/edit oAuth app, with callback url
+    `http://localhost:3001/api/oauth/callback`
 
-```bash
-npm run dev
-# or
-yarn dev
-```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+1.  `docker-compose build`
+1.  `docker-compose -f docker-compose.nginx.yml -f docker-compose.override.yml up backend`
```

Comment:
Should be `docker-compose -f docker-compose.nginx.yml -f docker-compose.override.yml -f docker-compose.yml up` accoring to all of the mentions of `-f`

Reply:
Seems like the override should be last, right? Otherwise i think it overwrites env vars
`docker-compose -f docker-compose.nginx.yml  -f docker-compose.yml  -f docker-compose.override.yml up~